integration/raft/config_test.go

```diff
@@ -617,11 +613,13 @@ var _ = Describe("EndToEnd reconfiguration and onboarding", func() {
 			}
 
 			By(fmt.Sprintf("Rotating cert on leader %d", leader))
+			time.Sleep(5 * time.Second)
```

Comment:
Please use Eventual with condition why are you waiting here instead of sleep.

Reply:
fixed